src/utils/makeReactNativeConfig.js

```diff
@@ -66,7 +66,7 @@ const getDefaultConfig = ({
    * `cheap-module-source-map` is faster than `source-map`,
    * but it doesn't have column mappings
    */
-  devtool: dev ? 'cheap-module-source-map' : 'source-map',
+  devtool: dev ? 'eval-source-map' : 'source-map',
```

Comment:
And what should the alternative be? `source-map` does not work, neither `cheap-eval-source-map`